src/runtime-rs/config/configuration-clh.toml.in

```diff
@@ -0,0 +1,352 @@
+# Copyright (c) 2019-2022 Alibaba Cloud
```

Comment:
Possibly bike-shedding, but I'm in two minds about the `clh` part of the filename:

- `clh` matches the golang filename convention of referring to Cloud Hypervisor as `clh`, but the rust runtime internally mostly now uses `ch`.
- `clh` also matches the name of the hypervisor used by the golang runtime (`[hypervisor.clh]`), but with the rust runtime we're now using `[hypervisor.cloud-hypervisor]` for more clarity.

In summary, I'm wondering if we should call it `configuration-cloud-hypervisor.toml.in` since that would

- Match the contents of the file (the `hypervisor` name).
- Make it clear to users / packagers that this file is not of the same format as the golang runtime config.

Reply:
With the same spirit, if we're going to make it `cloud-hypervisor` we should follow the same pattern on the makefile vars as well.

Reply:
@jodh-intel I am still in two minds about this one. Also depends on if we eventually converge on a single format for the config. For now, I have renamed the `configuration-clh.toml.in` file to `configuration-cloud-hypervisor.toml.in`.  I have held off on renaming the variables in the makefile as these are not really user-facing.

Reply:
I've just realised that we **cannot** to use `clh` for this version of the CH config file since when it is packaged, we need a unique name as all the config files end up in the same `./opt/kata/share/defaults/kata-containers/` in the archive.

Hence, `cloud-hypervisor` wfm :smile: